utilities/console.py

```diff
@@ -45,33 +52,33 @@ def connect(self):
 
         self._connect()
 
-        return self.child
+        return self.expecter
```

Comment:
Do we need to rename? `expecter` I understand the meaning, but so does `child` and it is heavily used in pexpect.

Reply:
`child` has no meaning to me, especial what a "child" of a console means, that is why I changed it.
This is the backend raw tool pexpect uses, returned by `pexpect.spawn`.

Reply:
I understand that. With the context to pexpect.spwan, it is more meaningful to me than expecter though.

Reply:
I do not understand. The attribute is of the `Console` class (its context), not of the `pexpect`. As someone who creates a console instance object, "child" has no meaning when seeing something like this: `console.child`.

But if you insist, I will just revert the name change. If it is clearer to you, no point to change it.